changelog/fix-6961-setting-options-non-prod-envs

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: dev
+
+Adds WCPay options to Woo Core option allow list to avoid 403 responses from Options API when geting and updating options in non-prod env.
```

Comment:
There is a small typo on `getting`.

```suggestion
Adds WCPay options to Woo Core option allow list to avoid 403 responses from Options API when getting and updating options in non-prod env.
```

Reply:
Thank you for catching that! I've fixed it in https://github.com/Automattic/woocommerce-payments/pull/7483/commits/cd207d1c6677155aedb511089a175bf4f359d04d